src/components/header_menu.jsx

```diff
@@ -61,13 +61,15 @@ const HeaderMenu = ({
             <strong>{filename}</strong>
           </Typography>
 
-          <WithBreakpoints hideOn={Breakpoints.Mobile}>
-            <NotePath
-              drivePath={drivePath}
-              path={file.attributes.path || simplifiedDrivePath}
-              target="_blank"
-            />
-          </WithBreakpoints>
+          {fileWithPath?.path && (
+            <WithBreakpoints hideOn={Breakpoints.Mobile}>
+              <NotePath
+                drivePath={drivePath}
+                path={fileWithPath.path}
+                target="_blank"
+              />
+            </WithBreakpoints>
+          )}
```

Comment:
I don't know if this is the right fix. 

What are you displaying if you share by link a note?
What are you displaying if you access a not inside a sub sub shared folder?